Monika After Story/game/script-topics.rpy

```diff
@@ -3180,8 +3180,7 @@ label monika_love:
 
     elif persistent._mas_monika_lovecounter == 40:
         m 1dubsu "Ehehe~"
-        m 1hubfa "I..."
-        m 1hubfb "Looooooooove you too, [player]!"
+        m 1hubfb "I...{wait=0.5} Looooooooove you too, [player]!"
```

Comment:
You can just have "w" instead of "wait". 

I might prefer the way it was before because of the different mouths. The open mouth emphasizes the "love" a little more while the "I" has a closed mouth.